currency_rate_update/models/currency_rate_update.py

```diff
@@ -128,6 +128,7 @@ def _selection_service(self, *a, **k):
     def refresh_currency(self):
         """Refresh the currencies rates !!for all companies now"""
         rate_obj = self.env['res.currency.rate']
+        today_dt = fields.Date.from_string(fields.Date.context_today(self))
```

Comment:
@alexis-via Why not `datetime.today()`?

Reply:
@alexis-via  I don't understand why `today_dt` is computed according to user timezone but in the same time the run method https://github.com/OCA/account-financial-tools/pull/724/files#diff-8e3e558fda291236aef336860358c181R214 makes a query without taking care of the user timezone. 
I've the feeling that we should ignore TZ into the context of these methods. May be I miss something.

Reply:
yes, you are right, I changed than.